my-app/src/components/ChangeInfo/index.jsx

```diff
@@ -1,11 +1,8 @@
 import EditIcon from "@mui/icons-material/Edit";
-import changeInfo from "../../changeInfo";
 
 const ChangeInfo = () => {
   return (
-    <div className="change-data"
-      onClick = {() => changeInfo()}>
-
+    <div className="change-data">
       <EditIcon style={{color: "black"}} />
 
       <span>Изменить</span>
```

Comment:
As option: you could use special variables/dictionaries for UI captions/warnings/messages/etc. So that you'll be able to change text quickly, and then will be ready to add i18n to your app (consider this variant)